SEED-platform/seed
# Use the same column for the names used in Property Insights

#### Any background context you want to provide?

#### What's this PR do?

#### How should this be manually tested?
Confirm that the tooltip in the chart and the table below the chart both use the display field from the organization settings.

#### What are the relevant tickets?
Fixes #3555

#### Screenshots (if appropriate)
